### DISCUSSION TITLE
[Feature]: May I get coords of text without paddings?

### DISCUSSION THREAD

**Discussion by Retryno** [fabricjs/fabric.js#9183] [Q&A] [answered]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have searched and referenced existing issues, feature requests and discussions
- [X] I am filing a **FEATURE** request.

### Description

So I need to get coordinates of text without paddings(red square) but for example aCoords, lineCoords and etc get back coords with paddings (blue square) . 
![image](https://github.com/fabricjs/fabric.js/assets/36660457/83cb67f9-1550-48f0-9a42-85dcb0714e27)

So qs maybe someone has any idea how to do it? Maybe i can`t see methods which can help me?

### Current State

I am trying to get coords of text in FabricJS without paddings and make the same image on my server using [Imagick](https://www.php.net/manual/ru/class.imagick.php) by coords from fabricjs, but I don`t know how to get information about paddings, because every time when I try to scale the paddings become bigger in progressive.

With image I do not have any problem and looks perfect, because image don`t have aditional paddindgs like text...
Example:
Fabricjs
![image](https://github.com/fabricjs/fabric.js/assets/36660457/06cd71d5-9658-4260-a917-4219f07ace5a)

And Imagic by coords from fabric
![image](https://github.com/fabricjs/fabric.js/assets/36660457/a00dc100-5ca4-48b0-aa02-186330ec23db)

### Additional Context

Hope you can help me, because I stopped on this moment a long time.

Thanks.

**Comment by ShaMan123** [comment, marked as answer]
1. override the relevant method (dig into `getCoords`) or set padding to 0 and back when calling `getCoords` while making sure to recalc the coods
2. Did you try using fabric in node?

**Reply by Retryno** [reply]
Thanks for answear!
Padding already 0,  maybe problem is in ImagikPhp... idk, but when I create Text with fontSize for example 334px and the same text on ImagikPhp 334px (line height and etc the same) height on output like fabric = 377px, Imagik = 391px and width the same...

Firstly I thought that this is canvas added new padding but now I see maybe problem is not here....

No I didn`t use Fabric in node, because the server side on php and ImagilPhp generate good quality of image.

**Comment by ShaMan123** [comment]
If this is related to the specific font we have nothing to do about it I believe
